README.md

```diff
@@ -1,62 +1,109 @@
+
+### Run with Docker
+
+The simplest way to run this service is to do so via Docker. A public [Docker image is available on Docker Hub](https://hub.docker.com/layers/mattermost-plugin-exchange-ews-proxy/brightscout/mattermost-plugin-exchange-ews-proxy/latest/images/sha256-f1172b134258a2bf9d75231ba4c7fd92c8161f7a4ffe4ec16adb3ea74b977e5c?context=explore).
+
+**Pull the image:**
 
-### Building the plugin
+```bash
+docker pull brightscout/mattermost-plugin-exchange-ews-proxy:latest
+```
+
+**Run the container:**
 
-- Run the following commands to prepare a compiled, distributable plugin jar:
 ```bash
-$ git clone git@github.com:Brightscout/mattermost-plugin-exchange-ews-proxy.git
-$ cd mattermost-plugin-exchange-ews-proxy
-$ ./gradlew build
+docker run --rm \
+  -p 8080:8080  \
+  --env-file=<path to env file> \
+  brightscout/mattermost-plugin-exchange-ews-proxy:latest
 ```
-- This will produce a `.jar` file in `/build/libs` directory that can be used to run server.
 
-### Environment variables
+### Run with Java
+
+To run the JAR file directly on a host, do the following:
 
-Create a `.env` file in the root folder and add the following environment variables:
+1. Ensure you have JDK 6 or later installed.
+2. Go to the [releases page of this GitHub repository](https://github.com/Brightscout/mattermost-plugin-exchange-ews-proxy/releases) and download the JAR file from the latest release.
+3. Ensure that the environment variables are exported to the shell.
+4. Start the server
 
-* **EWS_USERNAME** : Username of the Exchange service account.
-* **EWS_PASSWORD** : Password of the Exchange service account.
-* **EWS_DOMAIN** : Domain of the Exchange server.
-* **EWS_EXCHANGE_SERVER_URL** : URL of the Exchange server.
-* **EWS_SECRET_AUTH_KEY** : Key used for authentication between mattermost-plugin-exchange-ews-proxy and mattermost-plugin-exchange-mscalendar.
-* **JAR_PATH** : Path of the jar file that will be used to create docker image.
-* **SERVER_PORT** : Port on which the server will run.
-* **SUBSCRIPTION_LIFETIME_IN_MINUTES** : Expiry time of the subscription to push notifications in minutes.
+    ```bash
+     java -jar mattermost-plugin-exchange-ews-proxy-<version>.jar
+     ```
+
+## Development
+
+Make sure you have the following components installed:
+  - [JDK 6+](https://openjdk.java.net/install/) or later
+  - [Gradle](https://gradle.org/install/)
+  - [Docker](https://docs.docker.com/engine/install/)
+  - Make
 
-**Note**: When trying to run using java command, make sure to export these variables in the terminal.
-![image](https://user-images.githubusercontent.com/85667960/160066438-94fe5dd8-d020-4027-8318-8067607b1e2e.png)
+### Build and Run the JAR
 
-## Getting started
+Run the following commands to prepare a distributable JAR:
 
-#### Try it out with Java
+```bash
+git clone git@github.com:Brightscout/mattermost-plugin-exchange-ews-proxy.git
+cd mattermost-plugin-exchange-ews-proxy
+./gradlew build
```

Comment:
When we build using `java` we need the environment variables exported to the shell. So should we mention that step here?

Reply:
Added text for this.